pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -105,16 +105,16 @@ in rec {
       mkdir $out/include
       cp -rd ${llvmPackages.libcxx.dev}/include/c++     $out/include
 
-      ${lib.optionalString targetPlatform.isAarch64 ''
-        # copy .tbd assembly utils
```

Comment:
I don't think we need the TBD tools in the bootstrap. I could be wrong but I didn't need them [here](https://github.com/NixOS/nixpkgs/commit/96e64d04ffecd29c7f657d08fbf04e636668e86c). Maybe we can test, since Sandro is raising concerns about adding things to the bootstrap-tools?

Reply:
For what it's worth, I intended for `rewrite-tbd` to replace `print-reexports`. `print-reexports` only prints the list of re-exported things and uses shell to implement whatever logic is required. `rewrite-tbd` does mapping of exact matches and prefixes as well as checking that everything referred to is in the nix store.

Reply:
@thefloweringash, was this changed somewhere between April and now? Or is the current situation that both are needed? In case of the former we should probably drop the print-reexports commit again and a rebase on my end should make it unnecessary?

Reply:
Sorry, I should have been clearer. There's no change yet! The 10.12 sdk uses `print-reexports` and the 11.0 sdk (which came later) uses `rewrite-tbd`. I opened #182094 to share my thoughts on removing `print-reexports`.